addons/payment/views/payment_templates.xml

```diff
@@ -65,13 +65,13 @@
                         <ul class="pull-right list-inline payment_icon_list">
                             <t t-set="i" t-value="0"/>
                             <t t-foreach="acq.payment_icon_ids" t-as="pm_icon">
-                                <li t-att-class="'' if (i &lt; 3) else 'hidden'">
+                                <li t-att-class="'' if (i &lt; 6) else 'hidden'">
```

Comment:
Programming note: shouldn't we define a variable in the template like MAX_BRAND_ICONS and use it ? 

Side note: I think this commit ('up the number of brand card icon') could be merged with the other commits about brand icons.